desktop/apps/auctions2/templates/index.jsx

```diff
@@ -1,35 +1,9 @@
-import PropTypes from 'prop-types'
 import React from 'react'
-import { withLayout } from 'desktop/components/react/main_layout'
-
-function Index (props) {
-  const {
-    Layout: { Header, Body, Footer },
-    Templates: { Meta }
-  } = props
 
+export default function Index (props) {
```

Comment:
I wonder if `Index` is a great name for this. Would it make sense to use a more descriptive name, like maybe "AuctionsPage" or "AucitonsPageBody"? I guess that brings up the same debate from Slack on what a top-level component should be called.